Makefile

```diff
@@ -8,10 +8,11 @@ HUGO_OPTS=--disableFastRender
 # those from this repository
 prepare-preview: clean
 	mkdir -p $(PREVIEW_DEST)
-	git clone https://github.com/scientific-python-specs/specs.scientific-python.org $(PREVIEW_DEST)
+	git clone https://github.com/scientific-python/scientific-python.org $(PREVIEW_DEST)
```

Comment:
would this mean we don't need e.g. the netlify config here, e.g. setting and keeping the HUGO version in sync with the main website repo?

Reply:
No, I think this is just to prepare a scaffolding website for building the SPECs in.
At some point we thought to move that website to a new org, but that's no longer happening, so just using `scientific-python/scientific-python.org` instead.

Reply:
No, this is just a clean up from an experiment I did that I forgot about.